diff --git a/chrome/common/pref_names.cc b/chrome/common/pref_names.cc
index 032c0ca4c781c042df5eb7bb33b2b9f5c57e1008..c3014d9c015b71fabea0dfd64b5bcc37b1579945 100644
--- a/chrome/common/pref_names.cc
+++ b/chrome/common/pref_names.cc
@@ -1101,6 +1101,9 @@ const char kImportDialogBookmarks[] = "import_dialog_bookmarks";
 const char kImportDialogHistory[] = "import_dialog_history";
 const char kImportDialogSavedPasswords[] = "import_dialog_saved_passwords";
 const char kImportDialogSearchEngine[] = "import_dialog_search_engine";
+const char kImportDialogCookies[] = "import_dialog_cookies";
+const char kImportDialogStats[] = "import_dialog_stats";
+const char kImportDialogLedger[] = "import_dialog_ledger";
 
 // Profile avatar and name
 const char kProfileAvatarIndex[] = "profile.avatar_index";
